mloop/controllers.py

```diff
@@ -53,8 +54,27 @@ def create_controller(interface,
     elif controller_type=='random':
         controller = RandomController(interface, **controller_config_dict)
     else:
-        log.error('Unknown controller type:' + repr(controller_type))
-        raise ValueError
+        # If `controller_type` doesn't match any of the built-in controllers,
+        # looks for an external controller with matching name.
+        # The `controller_type` should be "module_name:controller_name".
+        parts = controller_type.split(":")
+        if (len(parts)==2) and all([len(part)>0 for part in parts]):
+            try:
+                module = import_module(parts[0])
```

Comment:
If the constructor is in a sub-module, can we do something like `qctrlmloop.submodule:QctrlController`?

Reply:
Yes, I think that should work – I tested it with `mloop.controllers:RandomController` and it seemed fine